### DISCUSSION TITLE
Select input filed with no padding

### DISCUSSION THREAD

**Discussion by gabrielaHashdex** [pratik227/quasar-qgrid#132] [General]
I'm encountering an issue where padding isn't being applied to select input elements. When I place multiple <select> fields side by side, there's no padding between them, causing them to appear directly adjacent to each other without any space.

q-field row no-wrap items-start q-field--filled q-input q-field--dense q-pl-xs q-pr-xs
![image](https://github.com/pratik227/quasar-qgrid/assets/105249281/9995c3d5-fcf8-40e8-9cfc-4ec0ad0d673c)
![image](https://github.com/pratik227/quasar-qgrid/assets/105249281/03859c94-55de-4f34-a193-2c0508ad94c9)

**Comment by pratik227** [comment]
Can you share the screenshot ?
